modules/ducktests/tests/ignitetest/services/utils/templates/cache_macro.j2

```diff
@@ -22,10 +22,20 @@
                 {% for cache in caches %}
                 <bean class="org.apache.ignite.configuration.CacheConfiguration">
                     <property name="name" value="{{ cache.name }}"/>
+                    <property name="cacheMode" value="{{ cache.cache_mode or 'PARTITIONED' }}"/>
                     {% if cache.cache_mode == 'PARTITIONED' %}
-                        <property name="backups" value="{{ cache.backups or 0 }}"/>
+                    <property name="backups" value="{{ cache.backups or 0 }}"/>
                     {% endif %}
                     <property name="atomicityMode" value="{{ cache.atomicity_mode or 'ATOMIC' }}"/>
+                    {% if cache.indexed_types %}
+                    <property name="indexedTypes">
```

Comment:
should this be in a separate file?

Reply:
what for?
this is part of the cache config